spec/api/v1/requests/users_request_spec.rb

```diff
@@ -243,4 +243,166 @@
       }.to raise_error(ActiveRecord::RecordNotFound)
     end
   end
+
+  describe 'update' do
+    it 'updates a user' do
+      user_details = {
+        email: 'ex@ample.com',
+        date_of_birth: '2001/02/03',
+        name: 'Jane Doe',
+        password: 'password'
+      }
+      headers = {"CONTENT_TYPE"  => 'application/json'}
+      post '/api/v1/users', headers: headers, params: JSON.generate(user_details)
+      created_user = User.last
+      expect(response.status).to eq 201
+      expect(created_user.email).to eq 'ex@ample.com'
+      expect(created_user.date_of_birth.to_s).to eq '2001-02-03'
+      expect(created_user.name).to eq 'Jane Doe'
+      expect(created_user.obituary).to eq nil
+      expect(created_user.password).to eq nil
+      expect(created_user.password_digest).is_a? String
+
+      updated_user_details = {
+        obituary: 'tedious and brief'
```

Comment:
Thanks for making it such that we only have to update one property at a time! This will make it easy for us to have a reusable function to do this